homeassistant/components/sun.py

```diff
@@ -30,6 +30,10 @@
 
 STATE_ATTR_AZIMUTH = 'azimuth'
 STATE_ATTR_ELEVATION = 'elevation'
+STATE_ATTR_NEXT_DAWN = 'next_dawn'
+STATE_ATTR_NEXT_DUSK = 'next_dusk'
+STATE_ATTR_NEXT_MIDNIGHT = 'next_midnight'
```

Comment:
Are these really useful? I am not a big fan of adding things because we can. There need to be use cases.